applications/ULFapplication/test_exemples/dam_2d.gid/dam_2d_0.230433928733.post.vv

```diff
@@ -0,0 +1,96 @@
+BeginZE /home/pavel/Kratos/applications/ULFapplication/test_exemples/dam_2d.gid/dam_2d_0.230433928733.post.vv
```

Comment:
I would add the vv files to gitignore....

Reply:
removed all the old test examples for the moment